lib/composables/Table.ts

```diff
@@ -79,6 +79,7 @@ export interface TableCellText extends TableCellBase {
   icon?: any
   value?: string | ((value: any) => string)
   link?(value: any, record: any): string
+  callback?(value: any, record: any): void
```

Comment:
Should we name this `onClick`? Since that's what we've been using in other places 👀